sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/EditUserSteps.java

```diff
@@ -0,0 +1,224 @@
+          softly
+              .assertThat(user.getRegion())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getRegion());
+          softly
+              .assertThat(user.getDistrict())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getDistrict());
+          softly
+              .assertThat(user.getCommunity())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getCommunity());
+          softly
+              .assertThat(user.getFacilityCategory())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getFacilityCategory());
+          softly
+              .assertThat(user.getFacilityType())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getFacilityType());
+          softly
+              .assertThat(user.getFacility())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getFacility());
+          softly
+              .assertThat(user.getFacilityNameAndDescription())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getFacilityNameAndDescription());
+          softly
+              .assertThat(user.getStreet())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getStreet());
+          softly
+              .assertThat(user.getHouseNumber())
+              .isEqualTo(CreateNewUserSteps.user.getHouseNumber());
+          softly
+              .assertThat(user.getAdditionalInformation())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getAdditionalInformation());
+          softly
+              .assertThat(user.getPostalCode())
+              .isEqualTo(CreateNewUserSteps.user.getPostalCode());
+          softly
+              .assertThat(user.getCity())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getCity());
+          softly
+              .assertThat(user.getAreaType())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getAreaType());
+          softly
+              .assertThat(user.getGpsLongitude())
+              .isEqualTo(CreateNewUserSteps.user.getGpsLongitude());
+          softly
+              .assertThat(user.getGpsLatitude())
+              .isEqualTo(CreateNewUserSteps.user.getGpsLatitude());
+          softly
+              .assertThat(user.getGpsAccuracy())
+              .isEqualTo(CreateNewUserSteps.user.getGpsAccuracy());
+          softly
+              .assertThat(user.getUserName())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getUserName());
+          softly.assertThat(user.getUserRole()).isEqualToIgnoringCase(role);
+          softly
+              .assertThat(user.getLimitedDisease())
+              .isEqualToIgnoringCase(CreateNewUserSteps.user.getLimitedDisease());
+          softly.assertAll();
+        });
+
+    Then(
+        "^I check the edited data is correctly displayed on Edit User page$",
+        () -> {
+          final User editUser = collectUserData();
+          softly
```

Comment:
can we use Truth and assert 2 objects to reduce these lines? :D

Reply:
updated